src/components/MapMarker/MapMarker.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RestaurantIcon from '../RestaurantIcon/RestaurantIcon';
+import './MapMarker.css';
+
+const MapMarker = (props) => {
+  const activeClass = props.active ? 'active' : '';
```

Comment:
you can also use this for conditional classnames: https://github.com/JedWatson/classnames
it has a syntax of:
let className = cx({
      map-marker: true,
      active: props.active,
    });